composer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "localgovdrupal/localgov_moderngov_tpl",
```

Comment:
I think we should call the module `localgov_moderngov` or `localgov_moderngov_template` and not have abbreviations such as `tpl` which is more of a D7 thing.

Reply:
Hmm.. not a bad suggestion but that entails lots of changes.  Any preference Finn? @finnlewis